lib/tests/test.py

```diff
@@ -0,0 +1,2 @@
+import time
+assert time.time() > 0
```

Comment:
Please give this file a more descriptive name, and maybe even a short comment to explain why it exists.